tasks.c

```diff
@@ -4434,7 +4434,9 @@ static void prvResetNextTaskUnblockTime( void )
          *
          * vTaskList() calls uxTaskGetSystemState(), then formats part of the
          * uxTaskGetSystemState() output into a human readable table that
-         * displays task names, states and stack usage.
+         * displays task: names, states, priority, stack usage and task number.

```

Comment:
Same comment as per prevision.

Reply:
Replaced with: Stack usage specified as the number of unused StackType_t words